From 97282b94bed23597571127532b144628834b3964 Mon Sep 17 00:00:00 2001
From: Jiangli Yuan <jlyuan@motorola.com>
Date: Fri, 4 Mar 2016 19:09:04 -0800
Subject: [PATCH 736/959] Revert "IKSWM-19916:mmc: map cmdq DCMD task desc page
 as DMA mapping."

This reverts commit 0dab967f1aeb80b8a8f9753ba2cc8472d3abfad8.

Signed-off-by: Jiangli Yuan <jlyuan@motorola.com>
Change-Id: Ibf1ecb0fb3f13c6adced57fda7e1fd5179fb6780
Reviewed-on: http://gerrit.mot.com/829471
SME-Granted: SME Approvals Granted
Submit-Approved: Jira Key <jirakey@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Reviewed-by: Sheng-Zhe Zhao <a18689@motorola.com>
Reviewed-by: Zhi-Ming Yuan <a14194@motorola.com>
---
 drivers/mmc/host/cmdq_hci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/cmdq_hci.c b/drivers/mmc/host/cmdq_hci.c
index 81062a7ca4d..fbb0117f96a 100644
--- a/drivers/mmc/host/cmdq_hci.c
+++ b/drivers/mmc/host/cmdq_hci.c
@@ -591,7 +591,7 @@ static void cmdq_prep_dcmd_desc(struct mmc_host *mmc,
 		mrq->cmd->opcode, timing, resp_type);
 	dataddr = (__le64 __force *)(desc + 4);
 	dataddr[0] = cpu_to_le64((u64)mrq->cmd->arg);
-	dma_map_single(mmc_dev(mmc), (void *)task_desc, cq_host->task_desc_len, DMA_BIDIRECTIONAL);
+
 }
 
 static int cmdq_request(struct mmc_host *mmc, struct mmc_request *mrq)
-- 
2.11.0

